xml/mdns_device-info_txt.xml

```diff
@@ -12,110 +12,462 @@
      The number specified after osxvers= is equivalent to the major
      version of OS X plus 4. E.g. osxvers=13 means OS X 10.9
    -->
+  <fingerprint pattern="^osxvers=17">
```

Comment:
Can we add Mojave while here?  Probably safe to assume it will be 18/14, right?

Reply:
Sounds reasonable.

Reply:
Done